server/apps/publish/archive_publish.py

```diff
@@ -670,6 +670,10 @@ def __init__(self, datasource=None, backend=None):
         super().__init__(datasource=datasource, backend=backend)
 
     def on_update(self, updates, original):
+        # check if we are trying to kill and item that is contained in normal non takes package
+        if is_item_in_package(original):
+            raise ValidationError(['This item is in a package' +
```

Comment:
This flavour of exception is displayed nicely by the UI, badRequestError is not so nice but will change it.